test/tstKDOP.cpp

```diff
@@ -38,8 +100,11 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(intersects_KDOP, KDOP_t, KDOP_types)
   BOOST_TEST(!intersects(x, KDOP_t{}));
 }
 
-BOOST_AUTO_TEST_CASE_TEMPLATE(intersects_box, KDOP_t, KDOP_types)
+BOOST_AUTO_TEST_CASE_TEMPLATE(intersects_kdop_box_3D, KDOP_t, KDOP_3D_types)
```

Comment:
Note to myself that the ordering kdop_box here is accurate even though at first sight it looked inconsistent with point_kdop below.